XPlat/TensorFlow.Lite/source/Xamarin.TensorFlow.Lite.Bindings.XamarinAndroid/Xamarin.TensorFlow.Lite.Bindings.XamarinAndroid.csproj

```diff
@@ -42,17 +42,17 @@
         <PackageReleaseNotes>
             Initial bindings
         </PackageReleaseNotes>
-        <PackageIconUrl></PackageIconUrl>
         <PackageTags>xamarin, android, bindings, google, tensorflow</PackageTags>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     </PropertyGroup>
 
     <ItemGroup>
         <TransformFile Include="transforms\*.xml" />
     </ItemGroup>
 
     <ItemGroup>
-        <LibraryProjectZip Include="..\..\externals\android\tensorflow-lite-1.9.0.aar">
-            <Link>Jars\tensorflow-lite-1.9.0.aar</Link>
+        <LibraryProjectZip Include="..\..\externals\android\tensorflow-lite-1.12.0.aar">
```

Comment:
What if we download this to just a `tensorflow-lite.aar` file and leave out the version so we don't have to bump it each time?

Reply:
changed